internal/build/phases.go

```diff
@@ -23,7 +23,6 @@ const (
 	appDir                    = "/workspace"
```

Comment:
Do we still need it?

Reply:
Thanks for the heads up. Will look into it.